PyPDF2/generic.py

```diff
@@ -2169,3 +2176,46 @@ def line(
             }
         )
         return line_obj
+
+
+def deprecate_bookmark(**aliases: str) -> Callable:
```

Comment:
Would `_utils` be a better place for this function since I'm assuming it's not meant to be public to end users?